modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v1/models/DataInputInfo.scala

```diff
@@ -0,0 +1,49 @@
+package org.ergoplatform.explorer.http.api.v1.models
+
+import io.circe.Codec
+import io.circe.generic.semiauto.deriveCodec
+import org.ergoplatform.explorer.db.models.aggregates.ExtendedDataInput
+import org.ergoplatform.explorer.{Address, BoxId, TxId}
+import sttp.tapir.{Schema, Validator}
+
+final case class DataInputInfo(
```

Comment:
We can simply reuse `InputInfo` as `DataInputInfo` would have the same set of fields.